libraries/AP_HAL_ChibiOS/hwdef/JFB110/defaults.parm

```diff
@@ -0,0 +1,47 @@
+# board setting
+BRD_VBUS_MIN 4.9
+
+# setup SERIAL0(OTG1) to Mavlink2
+SERIAL0_BAUD 1500
```

Comment:
I think we should remove the defaults for SERIAL0 because the regular AP value works fine.  I don't think changing the baud rate of a COM port actually makes any difference to the speed.

Reply:
I've removed serial0~serial3 in a follow-up commit.